### ISSUE TITLE
None of the type in colour logo is full black; below ACAG AAA

### ISSUE THREAD

**Issue by kevinmarks** [indieweb/branding#6]
I get emphasising WEB in the middle, but why is it #444444 and not black? The other type being grey #666666 is a contrast of at best 5.7 on white, which is below AAA WCAG at small sizes.
Why not #000000 for WEB and #4444444 for INDIE and CAMP

**Comment by veganstraightedge** [comment]
Because at full black, that part of the logo becomes the focal point in the visual hierarchy. And it shouldn't be. 

The contrast between INDIE/CAMP and WEB is only a nice to have, not a requirement. Think of it as a progressive enhancement. ;) 

In both cases (INDIE/CAMP and WEB) the contrast is high between it and the background (white, or black in the inverted logos).

**Comment by kevinmarks** [comment]
The contrast doesn't become high because you say so; this is a measurable thing. http://leaverou.github.io/contrast-ratio/#%23666-on-white

**Comment by karadaisy** [comment]
The guideline even says:

> Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.

The requirements for small point size body text do not seem applicable.

**Comment by veganstraightedge** [comment]
@kevinmarks Is 5.7 (#666 on white) not high? Or at least high enough?
